pkg/config/config_windows.go

```diff
@@ -23,10 +23,9 @@ import "github.com/containerd/containerd"
 // DefaultConfig returns default configurations of cri plugin.
 func DefaultConfig() PluginConfig {
 	return PluginConfig{
-		// TODO: JTERRY75 - Enable when we have CNI support.
 		CniConfig: CniConfig{
-			NetworkPluginBinDir:       "", // "E:\\containerd\\data\\cni\\bin",
-			NetworkPluginConfDir:      "", // "E:\\containerd\\data\\cni\\config",
+			NetworkPluginBinDir:       "C:\\containerd\\data\\cni\\bin",
```

Comment:
I think the defaults should be under "C:\Program Files\containerd\data". That is where the true defaults are and then we can override these in the config file if we choose.